content/_index.html

```diff
@@ -263,17 +263,19 @@
 	}(document, 'script', 'facebook-jssdk'));</script>
 <section>
 	<header>
-		<h1>Turning Hearts</h1>
+		<!-- <h1>Turning Hearts</h1> -->
+		<p></p>
 		<!-- <h2>Subtitle</h2> -->
 	</header>
```

Comment:
If you are going to deactivate the header then comment out the whole `<header>` block.